Source/ResponseSerialization.swift

```diff
@@ -32,8 +123,8 @@ extension Request {
 
         :returns: A string response serializer.
     */
-    public class func stringResponseSerializer(var encoding: NSStringEncoding? = nil) -> Serializer {
-        return { _, response, data in
+    public static func stringResponseSerializer(var encoding: NSStringEncoding? = nil) -> GenericResponseSerializer<String> {
```

Comment:
So it certainly could be, but my thought was that it shouldn't be. Here are a couple questions that I think will help us find the right balance:

* Does it really make sense to subclass the `data`, `string`, `JSON` or `propertyList` response serializers? 
* Does it even make sense to subclass `Request` at all?

> IMO the answer to both is NO.

The recommended usage here is certainly to extend `Request` to provide your own custom response serializations. You would implement your own `responseSomeCustomName` method that could either call one of the four default response serializers, or you would also create a `CustomResponseSerializer<T>` to handle your use case. 

Even in the event that you did end up subclassing `Request`, would it make sense to really override any of the four default response serializers with custom behavior? Wouldn't you instead implement your own generic response serializer and call your own custom response method? This was my reasoning for going with `static` rather than `class`.

Thoughts?